core-backend/common/src/funcs.rs

```diff
@@ -477,7 +474,7 @@ where
         gas: u64,
         err_mid_ptr: u32,
     ) -> Result<(), R::Error> {
-        ctx.run_fallible::<_, _, LengthBytes>(err_mid_ptr, RuntimeCosts::ReplyDeposit, |ctx| {
+        ctx.run_fallible::<_, _, ErrorBytes>(err_mid_ptr, RuntimeCosts::ReplyDeposit, |ctx| {
```

Comment:
Not related to this line, but `reply_deposit` is not covered with `#[host..]` macro. Why?